### DISCUSSION TITLE
Per-Organizer Stripe API Keys?

### DISCUSSION THREAD

**Discussion by melsabagh** [HiEventsDev/Hi.Events#334] [Q&A]
Attendize supported per-organizer Stripe keys. Does Hi.Events support this in any way? The variables mentioned in the documentation suggest that it only supports a single Stripe account.

**Comment by daveearley** [comment]
Hi @melsabagh,

Currently only a single Stripe account is supported. If there's enough demand I would not be against allowing a Stripe account per organizer.

**Comment by rodriwp** [comment]
Hello, I also need a stripe account per the organizer. I want to manage all events on my page, but then the money to go directly to the organizer so I don't need to handle any money ( I want to build a webpage to empower organizers that want to change how certain events are done, to connect participants with organizers with the same mindset). I'm a Platform Engineer, so I'm happy to also contribute as well.

**Comment by GeekTickets** [comment]
It would be great to be able to use stripe's connected accounts for a split payout.  So that the platform can keep the platform fees and the organiser gets the remainder of the ticket price.

**Comment by makifyegin** [comment]
Slightly different angle on my side — I'd like to scope invited team members to a specific organiser so they don't see the others on my account. Same underlying need: treating each organiser as its own silo.
